mkemlog.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * mkemlog: creates emlog files
+ *
+ * This utility can be used to create emlog files with out having
+ * to lookup the major number of /dev/emlog
+ *
+ * This code is freeware and may be distributed without restriction.
+ *
+ * James Rouzier <rouzier@gmail.com>
+ * July 7, 2016
+ *
+ */
+
+#define _BSD_SOURCE
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <error.h>
+#include <errno.h>
+
+#define EMLOG_DEVICE "/dev/emlog"
+#define USAGE "usage: mkemlog <filename> [size] [mode]"
```

Comment:
It would be great to provide a little-bit more verbose usage help text and an example.
 * maybe `logdevname` is a bit more descriptive than `filename`;
 * it would be great to mention that `size` is an integer meaning size of the log buffer in kilobytes (and thus no suffixes like K, M, G are supported).